credential/verification/verifiers.go

```diff
@@ -0,0 +1,94 @@
+package verification
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/TBD54566975/ssi-sdk/credential"
+	credschema "github.com/TBD54566975/ssi-sdk/credential/schema"
+	"github.com/pkg/errors"
+)
+
+const (
+	SchemaOption OptionKey = "schema"
+)
+
+var KnownVerifiers = []Verifier{
+	{
+		ID:         "Go Validation",
```

Comment:
instead of Go Validation maybe like Object Validation?

Reply:
yes, updated